codecov.yml

```diff
@@ -1,5 +1,5 @@
 coverage:
   status:
     project: off
-    patch: off
+    patch: on
```

Comment:
What does this control?

Reply:
I think it controls whether codecov makes the posts on PRs or not.